developer-workflow/psrt.rst

```diff
@@ -0,0 +1,148 @@
+
+* Once a CVE ID is assigned, the CVE ID and acceptance can be shared with the reporter.
+  Use this CVE ID for referencing the vulnerability. Ask the reporter
+  if they'd like to be credited publicly for the report and if so, how they'd like to be credited.
+  Add this information to the GitHub Security Advisory.
+
+* If the report is a vulnerability, determine the severity of the vulnerability. Severity is one of:
+  **Low**, **Medium**, **High**, and **Critical**. Coordinators can use their knowledge of the code, how the code is likely used,
+  or another mechanism like CVSS for determining a severity. Add this information to the GitHub Security Advisory.
+
+* Author the vulnerability advisory text. The advisory must include the following information:
+
+  * Title should be a brief description of the vulnerability and affected component
+    (for example, "Buffer over-read in SSLContext.set_npn_protocols()")
+
+  * Short description of the vulnerability, impact, and the conditions where the affected component is vulnerable, if applicable.
+
+  * Affected versions. This could be "all versions", but if the vulnerability exists in a new feature
+    or removed feature then this could be different. Include versions that are end-of-life in this calculation.
+    (for example, "Python 3.9 and earlier", "Python 3.10 and later", "all versions of Python")
+
+  * Affected components and APIs. The module, function, class, or method must be specified so users can
+    search their codebase for usage. For issues affecting the entire project, this can be omitted.
+
+  * Mitigations for the vulnerability beyond upgrading to a patched version, if applicable.
+
+  This can all be done within the GitHub Security Advisory UI for easier collaboration between reporter and coordinator.
+
+* Determine the fix approach and who will provide a patch. Some reporters are willing to provide or collaborate to create a
+  patch, otherwise relevant core developers can be invited to collaborate.
+
+* For **Low** and **Medium** severity vulnerabilities it is acceptable to develop a patch in public.
+  The pull request must be marked with the ``security`` and ``release-blocker`` labels so that a release
+  is not created without including the patch.
+
+* For **High** and **Critical** severity vulnerabilities the patch must be developed privately using GitHub Security Advisories'
+  "Private Forks" feature. Core developers can be added to the GitHub Security Advisory via "collaborators" to work
+  on the fix together. Once a patch is approved privately and tested, a public issue and pull request can be created
+  with the ``security`` and ``release-blocker`` labels.
+
+* Once the pull request is merged the advisory can be published. Send an advisory email to ``security-announce@python.org``
+  using the below template. Backport labels must be added as appropriate. After the advisory is published a CVE record
+  can be created.
+
+Template responses
+~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
------------------
```

Reply:
Resolved in https://github.com/python/devguide/pull/1348/commits/0344b7ce0b59c9ecebb41e28f353ed67ab704b30